build.gradle

```
@@ -1,6 +1,7 @@
 plugins {
     id "java"
     id "edu.wpi.first.GradleRIO" version "2024.1.1"
+    id("com.diffplug.spotless") version "6.23.3"
```

In general, it's good practice to try to keep the same syntax/style within the same file.  Pedantic, I know. :)

```suggestion
    id "com.diffplug.spotless" version "6.23.3"
```

---

Alright, I will remember this for the future. Thanks for letting me know.

The reason why I kept them was the newest spotless documentation, including the parentheses.